swarm/fuse/swarmfs_test.go

```diff
@@ -351,16 +344,18 @@ func (ta *testAPI) unmountWhenResourceBusy(t *testing.T) {
 
 	actualPath := filepath.Join(testMountDir, "2.txt")
```

Comment:
I'll make a separate PR for checking all the errors in swarm, otherwise all that will be mangled up in this cleanup PR.